test_xdb/models/under_test/clone_schema_grants_test.sql

```diff
@@ -0,0 +1,146 @@
+{{config({
+    "tags":["exclude_bigquery", "exclude_bigquery_tests"],
+    "pre-hook": [{"sql": "{%- if target.type == 'postgres' -%}
+                                DROP SCHEMA IF EXISTS schema_one CASCADE;
+                                DROP SCHEMA IF EXISTS schema_two CASCADE;
+                                DROP USER IF EXISTS user_1;
+                                DROP USER IF EXISTS user_2;
+                                DROP ROLE IF EXISTS role_1;
+                                DROP ROLE IF EXISTS role_2;
+                                CREATE SCHEMA schema_one;
+                                CREATE USER user_1;
+                                CREATE ROLE role_1;
+                                ALTER SCHEMA schema_one OWNER TO user_1;
+                                GRANT USAGE ON SCHEMA schema_one TO role_1;
+                                GRANT CREATE ON SCHEMA schema_one TO role_1;
+                                CREATE SCHEMA schema_two;
+                                CREATE USER user_2;
+                                CREATE ROLE role_2;
+                                ALTER SCHEMA schema_two OWNER TO user_2;
+                                GRANT USAGE ON SCHEMA schema_two TO role_2;
+                                GRANT CREATE ON SCHEMA schema_two TO role_2;
+                          {%- elif target.type == 'snowflake' -%}
+                                USE ROLE SYSADMIN;
+                                DROP SCHEMA IF EXISTS schema_one CASCADE;
+                                DROP SCHEMA IF EXISTS schema_two CASCADE;
+                                CREATE SCHEMA schema_one;
+                                GRANT ALL PRIVILEGES ON SCHEMA schema_one TO SYSADMIN;
```

Comment:
Switch this role to `xdb_role` instead of `sysadmin` and see if that works. CI runs as `xdb_role` against an isolated DB that it owns.

We should adjust the local workflow to account for the changes instead of changing the permissions for CI.

Reply:
It works! Thank you a lot!